draft-ietf-httpbis-variants.md

```diff
@@ -209,33 +214,32 @@ If the response can be used to satisfy more than one request), they can be liste
 
 ~~~ example
 Variants: Accept-Encoding;gzip;br, Accept-Language;en ;fr
-Variant-Key: gzip;fr, identity;fr
+Variant-Key: gzip;fr, "identity";fr
 ~~~
 
 indicates that this response can be used for requests whose Accept-Encoding algorithm selects "gzip" or "identity", as long as the Accept-Language algorithm selects "fr" -- perhaps because there is no gzip-compressed French representation.
 
-When more than one Variant-Key value is in a response, the first one present MUST indicate the variant-key for the response it occurs within.
```

Comment:
This requirement addresses an implementation constraint; if they can't easily link a response to multiple potential requests, this allows them to still behave correctly (if not optimally). Why did you remove it?

Reply:
The wording above doesn't reflect the intent well. It should be:

> When more than one Variant-Key value is in a response, the first one present MUST correspond to the request that caused that response to be generated.

Reply:
Ah, that makes sense. I've added your wording back.

I think it could be wordsmithed a little better now that Variant-Key is a structured header, but I don't want to delay merging the overall change. Specifically, "more than one Variant-Key value" implies to me that the Variant-Key header appears more than once, but we actually mean that the list parsed for Variant-Key from a response contains more than one inner-list, even if they're all parsed from a single instance of the header.